plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -937,84 +937,28 @@ protected TestTable createTableWithDoubleAndRealColumns(String name, List<String
     }
 
     @Test
+    @Override
```

Comment:
`TestDruidConnectorTest` has the similar test with a different name `testLimitPushDown` (d↔︎D). Can we remove the test now?